### PR TITLE
Add to_pil from_pil

### PR DESCRIPTION
**Pull request by GuoRentong**
Signed-off-by: GuoRentong <rt.guo@outlook.com>
#803

### CODE REVIEW

File: towhee/types/image_utils.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from towhee._types import Image
+import numpy as np
```
Comment (reiase) [inline towhee/types/image_utils.py:16]: numpy is not an official dependency.

File: towhee/types/image_utils.py [hunk 2]
```diff
@@ -32,3 +33,35 @@ def to_image_color(img: Image, target_mode: str):
...
+
+    Returns:
+        (`towhee.types.Image`)
+            The image wrapepd as towhee Image.
+    '''
+    return Image(np.array(pil_img), pil_img.mode)
+
+
+def to_pil(img: Image):
+    """
```
Comment (reiase) [inline towhee/types/image_utils.py:54]: this function should also support ndarray.

**Review by reiase — CHANGES_REQUESTED** [review]

File: towhee/types/image_utils.py (hunk 2 above)
Comment (filip-halt) [reply towhee/types/image_utils.py:56]: should we check if image can be converted or should we allow crash at runtime?

**Review by reiase — APPROVED** [review]

**Pull request merged**